extensions/markdown-basics/syntaxes/markdown.tmLanguage.json

```diff
@@ -1257,7 +1257,7 @@
 			]
 		},
 		"fenced_code_block_powershell": {
-			"begin": "(^|\\G)(\\s*)(`{3,}|~{3,})\\s*(?i:(powershell|ps1|psm1|psd1)((\\s+|:|,|\\{|\\?)[^`]*)?$)",
+			"begin": "(^|\\G)(\\s*)(`{3,}|~{3,})\\s*(?i:(powershell|ps1|psm1|psd1|pwsh)((\\s+|:|,|\\{|\\?)[^`]*)?$)",
```

Comment:
This file is actually comes from: https://github.com/Microsoft/vscode-markdown-tm-grammar

There's a config file that we use to generate the grammar. Can you submit the PR against that repo instead

Reply:
Done: https://github.com/microsoft/vscode-markdown-tm-grammar/pull/157

I don't actually care about the latex one if that isn't included, just saw it in search